test/Test/Toml/PrefixTree/Property.hs

```diff
@@ -5,7 +5,19 @@ import Hedgehog (forAll, (===))
 import Test.Toml.Gen (PropertyTest, genKey, genPrefixMap, genVal, prop)
 import Test.Toml.Property (assocLaw, identityLaw)
 
+import Data.String (IsString (..))
+
+import qualified Data.Text as Text
 import qualified Toml.PrefixTree as Prefix
+import qualified Toml.Printer as Printer
+
+----------------------------------------------------------------------------
+-- Key printing and parsing
+----------------------------------------------------------------------------
+test_KeyPrinting :: PropertyTest
+test_KeyPrinting = prop "Key printing: fromString . prettyKey == id" $ do
+  key <- forAll genKey
+  (fromString . Text.unpack) (Printer.prettyKey key) === key
```

Comment:
```suggestion
    (fromString . Text.unpack) (Printer.prettyKey key) === key
```

Reply:
Also, it's better to use `tripping` combinator from `hedgehog` library in such cases because it produces better error messages:

* https://hackage.haskell.org/package/hedgehog-0.6.1/docs/Hedgehog.html#v:tripping